apps/api_web/lib/api_web/controllers/prediction_controller.ex

```diff
@@ -122,11 +122,63 @@ defmodule ApiWeb.PredictionController do
     end
   end
 
-  defp include_legacy_stop_ids(ids, version) when version >= "2018-07-23" do
+  defp include_legacy_stop_ids(ids, version) when version >= "2019-02-12" do
     ids
   end
 
+  defp include_legacy_stop_ids(ids, version) when version >= "2018-07-23" do
+    include_legacy_stop_ids_2019_02_12(ids)
+  end
+
   defp include_legacy_stop_ids(ids, _version) do
+    ids
+    |> include_legacy_stop_ids_2019_02_12
+    |> include_legacy_stop_ids_2018_07_23
+  end
+
+  defp include_legacy_stop_ids_2019_02_12(ids) do
+    Enum.flat_map(
+      ids,
+      fn
+        "70061" ->
+          [
+            "70061",
+            "Alewife-01",
+            "Alewife-02"
+          ]
+
+        "70105" ->
+          [
+            "70105",
+            "Braintree-01",
+            "Braintree-02"
+          ]
+
+        "70036" ->
+          ["70036", "Oak Grove-01", "Oak Grove-02"]
+
+        "70001" ->
+          ["70001", "Forest Hills-01", "Forest Hills-02"]
+
+        "70199" ->
```

Comment:
Are you sure these IDs are right? 70199 is the E branch berth I think, which wasn't involved in these changes

Reply:
You're right: 70200 is the current Eastbound stop, so that's what I should be looking for.

Reply:
The Kenmore ones are correct:
- 70150 is Kenmore - Inbound, and 71150 is Kenmore - Inbound - B Branch
- 70151 is Kenmore - Outbound, and 71151 is Kenmore - Outbound - B Branch